.github/workflows/check.yml

```diff
@@ -201,7 +202,7 @@ jobs:
       with:
         submodules: true
     - name: configure
-      run: ./autogen.sh
+      run: ./autogen.sh --disable-wayland-security-context
```

Comment:
Why is this disabled in CI? Why is this a build time in the first place?

Reply:
Maybe the default should be "auto".

Reply:
I changed the default to "auto".